build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id "nebula.netflixoss" version "8.0.0"
     id "groovy"
+    id "nebula.dependency-lock" version "8.3.0" // keep this until nebula.netflixoss catches up to at least this version 
```

Comment:
I merged switch to plugin-plugin which has lock plugin included. This can be removed.